src/services/users.service.js

```diff
@@ -0,0 +1,50 @@
+const { createId } = require('../utils/id.service');
+
+let users = [];
+
+const resetUsers = () => {
+  users = [];
+};
+
+const getUsers = () => {
+  return users;
+};
+
+const getUser = (id) => {
+  return users.find((elem) => elem.id === id);
```

Comment:
Ensure that the `id` parameter is consistently treated as a number throughout the codebase. Consider using `+id` here to explicitly convert it to a number, as done in other parts of the code.